src/schema/rules/checks/fmap.yaml

```
@@ -49,3 +49,16 @@ MagnitudeFileWithTooManyDimensions:
     - nifti_header != null
   checks:
     - nifti_header.dim[0] == 3
+
+EPIWithBValsHasHighBVals:
+  issue:
+    code: EPI_WITH_BVALS_HAS_HIGH_BVALS
+    message: |
+      EPI field maps with b-values should not include high b-values (in essence, b-values > 100).
```

If we prefer to just require that there be some small b-values than all small:

```suggestion
EPISmallBVals:
  issue:
    code: EPI_WITH_BVALS_NEEDS_SMALL_BVALS
    message: |
      EPI field maps with b-values should include small (b<100) values.
```